docs/data/data-grid/primitives/toolbar/toolbar.md

```diff
@@ -0,0 +1,91 @@
+---
+title: React Data Grid - Toolbar component
+productId: x-data-grid
+components: ToolbarRoot, ToolbarButton
```

Comment:
This adds API links to the bottom of the page. Waiting on https://github.com/mui/material-ui/pull/44992 to be merged for the links to resolve correctly.